docs/contributors/getting-started.md

```diff
@@ -23,7 +23,7 @@ You will need the following applications installed:
   projects like [Metabrowse](https://github.com/scalameta/metabrowse).
 - `tests/input` example Scala code that is used as testing data for unit tests.
 - `tests/unit` moderately fast-running unit tests.
-- `tests/slow` slow integration tests.
```

Comment:
Let's keep the `slow` name of this project. The tests in this project are actually slow to run since they 

- require cross-building and `publishLocal` against 2.11/2.13
- run sbt/gradle/maven/mill/... `LspSuite` tests that take a long time to run
- run CPU intensive tests like indexing large classpaths

Reply:
Sounds good. I put the project name back to slow, and updated any other found references of `SlowSuite` that I found.